### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by Sharparam** [Palats/mapshot#33] [General]
For the fellow Arch users out there, I've created packages for it so mapshot can be installed easily:

 - [mapshot](https://aur.archlinux.org/packages/mapshot)
 - [mapshot-git](https://aur.archlinux.org/packages/mapshot-git)
 - [mapshot-bin](https://aur.archlinux.org/packages/mapshot-bin)

**Comment by Palats** [comment]
Thanks a lot! I've linked it from the readme / documentation.
